### PR TITLE
Internal init changes, add optinal MQ prefetch count 

### PR DESCRIPTION
**Pull request by EnoRage**

### CODE REVIEW

File: mq/mq.go [hunk 1]
```diff
@@ -67,11 +70,17 @@ func (q Queue) RunConsumer(consumer Consumer, cache storage.Addresses) {
 		return
 	}
 
+	if PrefetchCount < minPrefetchCount {
+		logger.Info("Change prefetch count to default")
+		PrefetchCount = defaultPrefetchCount
+	}
```
Comment (arein) [inline mq/mq.go]: How do you feel about `panic`ing here?

If not - consider to make it a `logger.warn` and to add the `PrefetchCount` and `minPrefetchCount` to the log statement so it's easier to fix

Comment (EnoRage) [reply mq/mq.go]: I though about panic, but if we will have a broken container on production with auto restart - it is worse than having small optimized prefetch count

Comment (EnoRage) [reply mq/mq.go]: Will change to warn

**Comment by EnoRage** [comment]
I will bring Docker tests for Observer, dangerous to make PRs without them

**Pull request merged**